File: 462. Minimum Moves to Equal Array Elements II/Python/Solution.py
class Solution(object):
    def minMoves2(self, nums):
        """
        :type nums: List[int]
        :rtype: int
        """
        nums.sort()
        result = 0
        i = 0
        j = len(nums) - 1
        while i < j:
            result += nums[j] - nums[i]
            i += 1
            j -= 1
        return result

File: 462. Minimum Moves to Equal Array Elements II/C++/Solution.cpp
class Solution {
public:
    int minMoves2(vector<int> &nums) {
        sort(nums.begin(), nums.end());
        int result = 0;
        int i = 0;
        int j = nums.size() - 1;
        while (i < j) {
            result += nums[j] - nums[i];
            i++;
            j--;
        }
        return result;
    }
};